packages/terra-date-picker/src/terra-dev-site/doc/example/CalendarFilter/CalendarFilterControlled.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import CalendarFilter from 'terra-date-picker/lib/CalendarFilter';
+import classNames from 'classnames/bind';
+import styles from '../DatePickerExampleCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The date value used for a controlled component.
+   */
+  value: PropTypes.string,
+};
+
+const defaultProps = {
+  value: undefined,
+};
+
+const CalendarFilterExample = (props) => {
```

Comment:
same destructure comment here, then we eliminate using `props`

Reply:
[updated](https://github.com/cerner/terra-framework/pull/1134/commits/4c5835bff1390ac82e351d16c9d8a36ec9206d6c)